u3-views/features/videoController.js

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
ฉันมีกฎ ESLint ที่แก้ไขปัญหานี้โดยอัตโนมัติ ;)
I have an ESLint rule that fixes this automatically ;)